File: app/src/main/java/rod/bailey/trafficatnsw/json/hazard/Period.java
package rod.bailey.trafficatnsw.json.hazard;

/**
 * Created by rodbailey on 11/10/2016.
 */
public class Period {
    public String closureType;
    public String direction;
    public String finishTime;
    public String fromDay;
    public String startTime;
    public String toDay;

    @Override
    public String toString() {
        StringBuffer buf = new StringBuffer( super.toString());

        buf.append("closeType="+ closureType);
        buf.append(",direction=" + direction);
        buf.append(",finishTime=" +finishTime);
        buf.append(",fromDay=" + fromDay);
        buf.append(",startTime="+ startTime);
        buf.append(",toDay="+ toDay);

        return buf.toString();
    }
}

File: app/src/androidTest/java/rod/bailey/trafficatnsw/json/hazard/Period.kt
package rod.bailey.trafficatnsw.json.hazard

/**
 * A period of time over which a hazard will be present (usually an event)
 */
data class Period (
	var closureType: String? = null,
	var direction: String? = null,
	var finishTime: String? = null,
	var fromDay: String? = null,
	var startTime: String? = null,
	var toDay: String? = null
)
